code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -23,6 +23,20 @@ God bless America.
 #define DEEPFRYER_COOKTIME 60
 #define DEEPFRYER_BURNTIME 120
 
+GLOBAL_LIST_INIT(deepfry_blacklisted_items, typecacheof(list(
+	/obj/item/screwdriver,
+	/obj/item/crowbar,
+	/obj/item/wrench,
+	/obj/item/wirecutters,
+	/obj/item/multitool,
+	/obj/item/weldingtool,
+	/obj/item/reagent_containers/glass,
+	/obj/item/reagent_containers/syringe,
+	/obj/item/reagent_containers/food/condiment,
+	/obj/item/storage,
```

Comment:
perhaps consider checking for the storage component instead of subtypes of /obj/item/storage, so that you wouldn't be able to bypass this check using a bible or the like?

Reply:
Bible is actually already a storage object, so it wouldn't be fried either.

Reply:
oh, huh.

still, just in case, we should check for the component, no?